actgen/utils.py

```diff
@@ -33,6 +34,8 @@ def get_common_arg_parser(self):
                             help='Which agent to use')
         parser.add_argument('--duplicate', '-d', type=int, default=5,
                             help='Number of times to duplicate actions')
+        parser.add_argument('--action_effect_multiplier', '-e', type=float, default=1,
+                            help='the action effect multiplier (use none-one values to enable semi-duplicate actions)')
```

Comment:
This help string could be a bit more descriptive. Maybe something like `'Multiplier in range [0,1] to create semi-duplicate actions from base actions.'`